Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -236,12 +236,6 @@ public func onNativeTouch(
     view: JavaObject?,
     touchParameters: JavaObject
 ) {
-    if env == nil || view == nil {
-        // We're in an invalid state where the JNI has exploded somehow
-        // Passing anything on to UIKit now would probably lead to a crash
-        return
-    }
```

Comment:
We added this to try to fix the crash, but adding it didn't actually improve anything and I don't believe this was ever the actual issue. Since the stack trace indicates that the crash indeed currently occurs when creating the SDL event, and not before it, we will see if we get crashes _not_ from the async `Task` in this function, which might indicate an issue with the jni functions here after all. I suspect that won't be the case.